projects/components/src/datagrid/filters/datagrid-numeric-filter.component.ts

```diff
@@ -26,15 +26,26 @@ export type DatagridNumericFilterConfig = FilterConfig<[number, number]>;
 })
 export class DatagridNumericFilterComponent extends DatagridFilter<[number, number], DatagridNumericFilterConfig>
     implements OnInit {
-    formGroup: FormGroup;
-
-    constructor(@Host() private filterContainer: ClrDatagridFilter, private fb: FormBuilder) {
+    constructor(@Host() filterContainer: ClrDatagridFilter, private fb: FormBuilder) {
         super(filterContainer);
         this.formGroup = this.fb.group({
             [FormFields.from]: null,
             [FormFields.to]: null,
         });
     }
+    formGroup: FormGroup;
+
+    /**
+     * Creates a {@link FilterRendererSpec} with the given config.
+     */
+    static factory(value?: [number, number]): FilterRendererSpec<DatagridNumericFilterConfig> {
```

Comment:
Can we rename this to create?

Also let's create a named type for this tuple. We should do that if we find ourselves repeating an inline type.

```typescript
type Range = [number, number]
```